docs/web3js-adapters.md

```diff
@@ -17,26 +17,39 @@ Then, you will have access to a bunch of helper methods to convert to and from W
 ```ts
 // For public keys.
 fromWeb3JsPublicKey(myWeb3JsPublicKey);
+fromWeb3JsPublicKeys([myWeb3JsPublicKey, myOtherWeb3JsPublicKey]);
 toWeb3JsPublicKey(myUmiPublicKey);
+toWeb3JsPublicKeys([myUmiPublicKey, myOtherUmiPublicKey]);
 
 // For keypairs.
 fromWeb3JsKeypair(myWeb3JsKeypair);
+fromWeb3JsKeypairs([myWeb3JsKeypair, myOtherWeb3JsKeypair]);
 toWeb3JsKeypair(myUmiKeypair);
+toWeb3JsKeypairs([myUmiKeypair, myOtherUmiKeypair]);
 
 // For transactions.
 fromWeb3JsTransaction(myWeb3JsTransaction);
+fromWeb3JsTransactions([myWeb3JsTransaction, myOtherWeb3JsTransaction]);
 toWeb3JsTransaction(myUmiTransaction);
+toWeb3JsTransactions([myUmiTransaction, myOtherUmiTransaction]);
 fromWeb3JsLegacyTransaction(myLegacyWeb3JsTransaction);
+fromWeb3JsLegacyTransactions([myLegacyWeb3JsTransaction, myOtherLegacyWeb3JsTransaction]);
 toWeb3JsLegacyTransaction(myUmiTransaction);
+toWeb3JsLegacyTransactions([myUmiTransaction, myOtherUmiTransaction]);
 
 // For transaction messages.
 fromWeb3JsMessage(myWeb3JsTransactionMessage);
+fromWeb3JsMessages([myWeb3JsTransactionMessage, myOtherWeb3JsTransactionMessage]);
 toWeb3JsMessage(myUmiTransactionMessage);
+toWeb3JsMessages([myUmiTransactionMessage, myOtherUmiTransactionMessage]);
 toWeb3JsMessageFromInput(myUmiTransactionInput);
+toWeb3JsMessagesFromInputs([myUmiTransactionInput, myOtherUmiTransactionInput]);
 
 // For instructions.
 fromWeb3JsInstruction(myWeb3JsInstruction);
+fromWeb3JsInstructions([myWeb3JsInstruction, myOtherWeb3JsInstruction]);
 toWeb3JsInstruction(myUmiInstruction);
+toWeb3JsInstructions([myUmiInstruction, myOtherUmiInstruction]);
```

Comment:
this is kinda ugly, so happy to do in a different way if you'd like, for ex just specifying in text that these functions all have plural equivalents